pairing-tool/index.js

```diff
@@ -13,12 +13,12 @@ csvWriter.pipe(fs.createWriteStream(outputFile))
 
 async function scanAndPairOneButton() {
 
-    let promise = new Promise((resolve, reject) => {
+    let promise = new Promise((resolve, reject) => {    // eslint-disable-line no-unused-vars
```

Comment:
@schwarrrtz 
What do you think about the disabling eslint inline for these function parameters that you wanted to keep?

Reply:
this is a good compromise - I've tweaked these config comments in a commit that I just pushed to this branch.